How-to-upgrade-review-version.md

```diff
@@ -0,0 +1,399 @@
+# How to Upgrade to TypeScript Without Anybody Noticing
+
+This guide will show you how to upgrade to Typescript without anybody
```

Comment:
Uppercase `S` in `Typescript`

Reply:
I disagree with this spelling and the more I use the other one, the more chance it has to become normative.

That's right. I moved on from being a prescriptivist linguist to being an *activist* linguist.